buildscripts/buildvars-setup.sh

```diff
@@ -244,6 +215,42 @@ while [ "$1" != "" ]; do
     shift
 done
 
+if [ $__BuildArch == "wasm" ]; then
+    export __BuildOS=WebAssembly
+else
+    # Use uname to determine what the OS is.
```

Comment:
As I mentioned in response to @jkotas's question, we'll probably need to capture the NuGet information and maybe the OS for getting the right .NET Core dependencies for ILC (but we probably need to use a different variable rather than ```__BuildOS```)

Reply:
I'm not 100% clear on what the final build/packaging story will be, so would welcome any suggestions here. 

I anticipated that ultimately any build would need include the WASM build so that the static runtime `.bc` file can be packaged/embedded somewhere it can be accessed by the current platforms ILC. 

The reason I went for this approach now, is it allows the WASM build to be attempted completely separately, until it is at a point where it actually works.

Reply:
I'd suggest starting with keeping the logic to set __NugetRuntimeId since that's required to publish ILC to the tools directory

Reply:
That sounds reasonable